hydroengine_service/dgds_functions.py

```diff
@@ -26,6 +26,11 @@
 LAND = ee.Image('users/gena/land_polygons_image')
 LANDMASK = ee.Image(LAND.unmask(1, False).Not().resample('bicubic').focal_mode(2))
 
+def validate_min_lt_max(min, max):
+    min_lt_max = min < max
+    if not min_lt_max:
```

Comment:
if min > max

bit overdoing it with defining  min lt max and then validating the other way around?

Reply:
Good point, will reduce